change/beachball-ef09dabd-ce24-4044-af18-daf358a4f64c.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Prettier correction",
```

Comment:
This is used in the changelog, so it should describe the actual change.

Reply:
updated